registration-checker-compiler-plugin/src/main/scala/org/virtuslab/ash/SerializerCheckCompilerPluginComponent.scala

```diff
@@ -27,6 +28,7 @@ class SerializerCheckCompilerPluginComponent(
     s"Checks marked serializer for references to classes found in $serializerCheckPhaseName"
 
   private val serializerType = typeOf[Serializer]
+  private val serializerRegexFilter = typeOf[SerializerRegexFilter]
```

Comment:
Hmmm what is the regex specifically matched against? Also, is it "includes" or "fully matches" semantics?

Reply:
Full match